### PR TITLE
LF 16.5

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Actions required to update
None

### 🎉 Features
[Callout Disclosure] New HTML & Angular component
[Callout Popup] New HTML & Angular component
[luButton] New Button directive https://github.com/LuccaSA/lucca-front/pull/2372
[Utility] New `u-displayContents` utility https://github.com/LuccaSA/lucca-front/pull/2376

### 🎨 Visual changes
[Radio Button / Button] Standardize box shadow (outset / grey 300) https://github.com/LuccaSA/lucca-front/pull/2387
[New Badge] Update color from Product to Lucca https://github.com/LuccaSA/lucca-front/pull/2391
[Navside/Select/Table/Vertical Navigation] Standardize expandable icon and add transition https://github.com/LuccaSA/lucca-front/pull/2398
[Card] Add focus style on clickable card https://github.com/LuccaSA/lucca-front/pull/2392

### 🚑 Fixes
[Date Select] Fix search with year granularity https://github.com/LuccaSA/lucca-front/pull/2381 
[Textfield] Fix right padding with search / S & XS sizes https://github.com/LuccaSA/lucca-front/pull/2386
[Icon] Fix height on Angular component https://github.com/LuccaSA/lucca-front/pull/2388
[Navside] Fix numericBadge scope https://github.com/LuccaSA/lucca-front/pull/2390
[Selects] Fix displayer issues https://github.com/LuccaSA/lucca-front/pull/2395
[Simple & Multi select] Components are now _touched_ when panel is closed https://github.com/LuccaSA/lucca-front/pull/2384
[Simple Select] URI encoding done twice instead of just once https://github.com/LuccaSA/lucca-front/pull/2399
[Select] Fix invalid initial v4 sort https://github.com/LuccaSA/lucca-front/pull/2371
Fix specificity issues generated by Angular components https://github.com/LuccaSA/lucca-front/pull/2403

### 🔨 Technical
[Numeric Badge] Improve nested compatibility with parents https://github.com/LuccaSA/lucca-front/pull/2389
[Textfield] Remove outdated CSS vars https://github.com/LuccaSA/lucca-front/pull/2394
Remove mod-M as default value on Angular components https://github.com/LuccaSA/lucca-front/pull/2401
[Icons] Remove SVG and TTF files https://github.com/LuccaSA/lucca-front/pull/2400
Chore(ci): use linux node https://github.com/LuccaSA/lucca-front/pull/2373

### 📄 Documentation
[Checkbox] Improve mixed story https://github.com/LuccaSA/lucca-front/pull/2396
[Inline message] Add stories https://github.com/LuccaSA/lucca-front/pull/2393
Improve NG components documentation https://github.com/LuccaSA/lucca-front/pull/2377
Improve HTML components documentation on zeroheight https://github.com/LuccaSA/lucca-front/pull/2375

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9433/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9450/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9462/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9514/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9598/ui-diff)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9614/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9616/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9660/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9663/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2408/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9665/ui-diff)

**Review by GuillaumeNury — APPROVED** [review]
TLDR
